src/Analyzer.Types/IEvaluation.cs

```diff
@@ -12,9 +12,9 @@ namespace Microsoft.Azure.Templates.Analyzer.Types
     public interface IEvaluation
     {
         /// <summary>
-        /// Gets the name of the rule this evaluation is for.
+        /// Gets the ID of the rule this evaluation is for.
         /// </summary>
-        public string RuleName { get; }
+        public string RuleID { get; }
```

Comment:
I think `RuleId` and `Id` (lowercase 'd') would be a better style, and are more inline with code style guidelines.